test/v1alpha1/resources/operator.knative.dev_v1alpha1_knativeserving_cr.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp":
         "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s",
         "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
+      logging.enable-request-log: "true"
```

Comment:
Is this useful in general or was this extra debug needed to fix an issue at hand ? Just to be sure that the noise/content ratio doesn't increase too much.

Reply:
It's needed for all tests to pass. Upstream has an API change here where the flag now needs to be set explicitly to true rather than it assuming that logs should be printed if just the template has been filled in.